curriculum/challenges/english/25-front-end-development/workshop-hotel-feedback-form/66a962954f4e0d76223b37ed.md

```diff
@@ -56,13 +56,13 @@ assert.strictEqual(document.querySelectorAll("fieldset:nth-of-type(3) label + in
 You should have a `label` element with the text of `Reputation` below the location checkbox.
 
 ```js
-assert.isNotNull(document.querySelectorAll('fieldset:nth-of-type(3) input[value="location"] + label')?.textContent, 'Reputation');
+assert.strictEqual(document.querySelectorAll('fieldset:nth-of-type(3) label')[3]?.textContent.trim(), 'Reputation');
 ```
 
 Your `label` should have the `for` attribute set to `"reputation"`.
 
 ```js
-assert.strictEqual(document.querySelector('fieldset:nth-of-type(3) input[value="location"] + label')?.getAttribute('for'), 'reputation');
+assert.strictEqual(document.querySelectorAll('fieldset:nth-of-type(3) label')[3]?.getAttribute('for'), 'reputation');
 ```
```

Comment:
out of scope, please revert, and if you want to propose changes more than adding `trim`, create a new PR